packages/api-key-service/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "api-key",
```

Comment:
Think this needs to be api-key-service to match the directory - it's to do with how yarn workspaces work

Reply:
Don't know why but this comment is duplicated!

Reply:
sure, I'll change it, did think this when I was watching the tests